packages/fury-adapter-swagger/test/fixtures/schema-references-example-invalid.yaml

```diff
@@ -0,0 +1,10 @@
+swagger: '2.0'
+info:
+  title: My API
+  version: 1.0.0
+paths: {}
+definitions:
+  Test:
```

Comment:
This 'Schema Object' declares an array type which has an sample value of an array. The array's first value is an object which has a key `$ref` and the value `#/info`, it is not a reference as per the spec and we are not treating it as one since it doesn't reference any valid component in the document.